apps/user-portal/src/components/linked-accounts/linked-accounts.tsx

```diff
@@ -18,41 +18,32 @@
 
 import React, { ChangeEvent, FunctionComponent, useEffect, useState } from "react";
 import { useTranslation } from "react-i18next";
-import {
-    Button,
-    Divider,
-    Form,
-    Grid,
-    Icon,
-    List
-} from "semantic-ui-react";
-import { addAccountAssociation, getAssociations } from "../actions/associated-accounts";
-import { SettingsSectionIcons } from "../configs";
-import { NotificationActionPayload } from "../models/notifications";
-import { EditSection } from "./edit-section";
-import { SettingsSection } from "./settings-section";
-import { UserImagePlaceHolder } from "./ui";
+import { Button, Divider, Form, Grid, Icon, List } from "semantic-ui-react";
+import { addAccountAssociation, getAssociations } from "../../api";
+import { SettingsSectionIcons } from "../../configs";
+import { Notification } from "../../models";
+import { EditSection, SettingsSection, UserImage } from "../shared";
 
 /**
- * Proptypes for the associated accounts component.
+ * Proptypes for the liked accounts component.
  */
-interface AssociatedAccountsProps {
-    onNotificationFired: (notification: NotificationActionPayload) => void;
+interface LinkedAccountsProps {
+    onNotificationFired: (notification: Notification) => void;
 }
 
 /**
- * Associated accounts component.
+ * Linked accounts component.
  *
- * @param {BasicDetailsProps} props - Props injected to the basic details component.
+ * @param {LinkedAccountsProps} props - Props injected to the component.
  * @return {JSX.Element}
  */
-export const AssociatedAccountsComponent: FunctionComponent<AssociatedAccountsProps> = (
-    props: AssociatedAccountsProps
+export const LinkedAccounts: FunctionComponent<LinkedAccountsProps> = (
+    props: LinkedAccountsProps
 ): JSX.Element => {
-    const [associations, setAssociations] = useState([]);
-    const [username, setUsername] = useState("");
-    const [password, setPassword] = useState("");
-    const [editingForm, setEditingForm] = useState({
+    const [ associations, setAssociations ] = useState([]);
+    const [ username, setUsername ] = useState("");
+    const [ password, setPassword ] = useState("");
+    const [ editingForm, setEditingForm ] = useState({
```

Comment:
Do we need these spaces between the square brackets and the characters?

Reply:
It looks a bit cleaner with the spaces IMO. We can have a discussion and decide on things like this and we could create a CONTRIBUTING guidelines file and specify these kind of practices.